Sweeper/View Controllers/CreatePinViewController.swift

```diff
@@ -80,33 +175,9 @@ class CreatePinViewController: UIViewController, UINavigationControllerDelegate
         }
     }
     
-    @IBAction func importImage(_ sender: Any) {
-        let alertController = UIAlertController(title: "Choose image", message: nil, preferredStyle: .actionSheet)
-        alertController.addAction(UIAlertAction(title: "Camera", style: .default, handler: { (action) in
-            self.showPicker(with: UIImagePickerControllerSourceType.camera)
-        }))
-        alertController.addAction(UIAlertAction(title: "Photo Library", style: .default, handler: { (action) in
-            self.showPicker(with: UIImagePickerControllerSourceType.photoLibrary)
-        }))
-        alertController.addAction(UIAlertAction(title: "Cancel", style: .default, handler: nil))
-        self.present(alertController, animated: true, completion: nil)
-    }
-    
     @objc func onCancel() {
         dismiss(animated: true, completion: nil)
```

Comment:
Do we want to check if there's a pin creation in progress, and put up a verification alert to exit?

Reply:
Yeah, we should do that.